webapp/sources/rudder/rudder-web/src/main/resources/demo-rudder-users.xml

```diff
@@ -33,11 +33,14 @@
 
 <!--
   The "authentication" tag should have a "hash" attribute, with these allowed values:
-    "md5", "sha1", "sha256", "sha-256", "sha512", "sha-512"
+    "bcrypt", "md5", "sha1", "sha256", "sha-256", "sha512", "sha-512"
 
   Not specifying the "hash" attribute means using plain text passwords, which is not
   adviseable for security reasons.
 
+  To create a bcrypt hash of your password, please run this command:
+    htpasswd -nBC 12 "" | tr -d ':\n' | sed 's/$2y/$2b/'
```

Comment:
Is the sed really necessary? The compare algo should work with "2y", and it's a risk that user do ad thing